### ISSUE TITLE
Generate Predictions

### ISSUE THREAD

**Issue by justjasongreen** [justjasongreen/predictivepunter#21]
Generate a list of predictions for a specified race, based on a range of regression and classification estimators found in the scikit-learn library.  For each estimator type, generate predictions when training/testing both with and without sample weights.

Each prediction should containing the following values:
- [x] Sample schema version
- [x] Predictor version
- [x] Estimator type (class name)
- [x] Uses sample weights
- [x] Ordered list of predicted runner placements
- [x] Confidence (r^2 score for regression estimators or accuracy for classification estimators)

Potentially suitable regression estimators include:
- [x] sklearn.ensemble.AdaBoostRegressor
- [x] sklearn.ensemble.BaggingRegressor
- [x] sklearn.ensemble.ExtraTreesRegressor
- [x] sklearn.ensemble.GradientBoostingRegressor
- [x] sklearn.ensemble.RandomForestRegressor
- [x] sklearn.linear_model.LinearRegression
- [x] sklearn.linear_model.LogisticRegression
- [x] sklearn.linear_model.Ridge
- [x] sklearn.linear_model.SGDRegressor
- [x] sklearn.svm.SVR
- [x] sklearn.svm.NuSVR
- [x] sklearn.tree.DecisionTreeRegressor
- [x] sklearn.tree.ExtraTreeRegressor

Potentially suitable classification estimators include:
- [x] sklearn.ensemble.AdaBoostClassifier
- [x] sklearn.ensemble.BaggingClassifier
- [x] sklearn.ensemble.ExtraTreesClassifier
- [x] sklearn.ensemble.GradientBoostingClassifier
- [x] sklearn.ensemble.RandomForestClassifier
- [x] sklearn.linear_model.RidgeClassifier
- [x] sklearn.linear_model.SGDClassifier
- [x] sklearn.svm.SVC
- [x] sklearn.svm.NuSVC
- [x] sklearn.tree.DecisionTreeClassifier
- [x] sklearn.tree.ExtraTreeClassifier
